prefq/examples/imitation_preference_comparisons.py

```diff
@@ -162,7 +163,7 @@ def _get_feedback_from_server(self):
         def _wait_for_feedback_request(url):
             while True:
                 time.sleep(5)
-                response = requests.get(url)
+                response = requests.get(url, timeout=5)
```

Comment:
Minor nit: This change does not belong into the "Unify Codebase for functionality across both examples".

Reply:
does this hold true, considering, that in query_client.py, the wait_for_feedback() function shows the same behavior? Unifying the Codebase I wanted to enforce the same behavior in both modules.

The precommit checks required a timeout in this module, but not in imitation_preference_comparisons.py (or I might have overseen this change in the interactive rebase & accidently discarted it)

https://github.com/timokau/prefq/pull/19/files#diff-4597eddb0345e4b99f750c587943b1ed1d4ea47f6dc312c1e439fcd48208eed8

Reply:
I thought the commits were about unifying the server code, since the clients don't necessarily need to have the exact same behavior. But you have a point, could be argued that this change does belong here :) Its not very important either way, I'll just close this.